openssl/src/crypto/rsa.rs

```diff
@@ -277,4 +303,22 @@ mod test {
 
         assert!(result);
     }
+
+    #[test]
+    pub fn test_password() {
```

Comment:
This will need to be flagged under the catch_unwind feature - I fixed the travis builds so if you rebase onto master it should be ok.